csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿global using tupleAliasGlobalUsing = (int num1, string str, int num2);
+global using System;
```

Comment:
If this is really needed, let's move it to the Usings.cs as well. It's conventional to have all global usings in one file called Usings.cs

Reply:
This PR has been updated to reflect this change.